addons/website_sale/static/src/js/website_sale.js

```diff
@@ -129,10 +132,16 @@ publicWidget.registry.websiteSaleCartLink = publicWidget.Widget.extend({
     /**
      * @private
      */
-    _updateCartQuantityValue() {
-        return this._rpc({route: "/shop/cart/quantity"}).then((cartQty) => {
-            this.cartQty = cartQty;
-        });
+    async _updateCartQuantityValue() {
+        if ('website_sale_cart_quantity' in sessionStorage) {
+            this.cartQty = sessionStorage.getItem('website_sale_cart_quantity');
```

Comment:
import from `@web/core/browser/browser` and use `sessionStorage` from that object instead of the one of the browser directly